BUILD

```diff
@@ -13,3 +13,17 @@ filegroup(
         "//os:srcs",
     ],
 )
+
+# For use in Incompatible Target Skipping:
+# https://docs.bazel.build/versions/master/platforms.html#skipping-incompatible-targets
+#
+# Specifically this lets targets declare incompatibility with some set of
+# platforms. See
+# https://docs.bazel.build/versions/master/platforms.html#more-expressive-constraints
+# for some more details.
+constraint_setting(name = "incompatible_setting")
+
+constraint_value(
+    name = "incompatible",
```

Comment:
I suppose calling this `none` could work too.

Do people have any cool ideas?

```python
target_compatible_with = select([
    "@platforms//os:linux": [],
    "//conditions:default": ["@platforms:none"],
]),
```

Reply:
I think I prefer `incompatible`, `none` feels like I should somehow be unsetting the target platform, not that it's impossible to match.